packages/components/src/__future__/Tag/Tag.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useMemo, FC, ReactNode, Children, isValidElement } from "react"
+import classNames from "classnames"
+import { v4 } from "uuid"
+
+const isElementText = (child: ReactNode): boolean =>
+  typeof child === "string" && child.replace(/\s/g, "").length > 0
+
+const isElementNumber = (child: ReactNode): boolean => typeof child === "number"
+
+const isElementIcon = (child: ReactNode): boolean =>
+  isValidElement(child) &&
+  typeof child.type !== "undefined" &&
+  typeof child.type !== "string" &&
+  typeof child.type !== "number" &&
+  child.type.name === "Icon"
+
+type TagColors =
+  | "gray"
+  | "blue"
+  | "green"
+  | "yellow"
+  | "orange"
+  | "red"
+  | "purple"
+
+export interface TagProps {
+  children: ReactNode
+  color?: TagColors
+}
+
+export const Tag: FC<TagProps> = ({ children, color = "gray" }) => {
+  const uuid = useMemo(() => v4(), [])
+
+  return (
+    <span
+      className={classNames(
+        "inline-flex items-center px-8 py-4 gap-4 rounded-[18px] font-family-paragraph text-paragraph-sm font-weight-paragraph leading-paragraph-sm whitespace-nowrap",
```

Comment:
The rounded needs to be `100%`

Reply:
Our TW preset has the `rounded-full` set to `100%` instead of `9999px`.
![Screenshot 2023-07-10 at 12 50 41 pm](https://github.com/cultureamp/kaizen-design-system/assets/4048660/a434f718-87eb-465d-a81c-4754f6edff5f)

Should I use something like `rounded-[100vmax]` instead?
![Screenshot 2023-07-10 at 12 53 59 pm](https://github.com/cultureamp/kaizen-design-system/assets/4048660/a4010c75-a95a-48e5-945b-8d7914e096ba)

Reply:
Hm I don't want to rely on a viewport unit for this, so I think just go with the height of the pill, `20px` and if we need to make it scale we can change it later.